internal/events/events.go

```diff
@@ -169,19 +171,22 @@ func (l *Logger) Unsubscribe(s *Subscription) {
 	l.mutex.Unlock()
 }
 
+// Poll returns an event from the subscription or an error if the poll times
+// out of the event channel is closed. Poll should not be called concurrently
+// from multiple goroutines for a single subscription.
```

Comment:
Nah... Goroutine-unsafe is the default, I'm just being unnecessarily picky in the comment.